### ISSUE TITLE
Expiry Grace Period

### ISSUE THREAD

**Issue by phelixbtc** [namecoin/nmcontrol#64]
Have NMControl bail on names that are about to expire.

https://forum.namecoin.info/viewtopic.php?p=10756#p10756

**Comment by JeremyRand** [comment]
Agreed, we should do this.  The question is what the period should be, and whether it should be variable based on the name owner's wishes.  I don't think there was a consensus on this.

Thoughts?
